### PR TITLE
fix: need to convert instance type from the JS class to the wrapper class

### PR DESCRIPTION
**Pull request by wf9a5m75**
Please merge this pull request, and release as `@ionic-native/google-maps` 4.2.2.
And please specify the dependency module `@ionic-native/core` 4.2.0

### CODE REVIEW

File: src/@ionic-native/plugins/google-maps/index.ts [hunk 1]
```diff
@@ -687,21 +663,53 @@ export class BaseClass {
    *
    * @return {Observable<any>}
    */
-  @CordovaInstance({
```
Comment (ihadeed) [inline src/@ionic-native/plugins/google-maps/index.ts]: I see that you've removed some decorators. I suggest you replace them with `CordovaCheck` and `InstanceCheck` decorators. These two decorators check if the plugin/objectInstance exists before executing the functionality. This helps eliminate runtime errors when we're developing outside of a Cordova platform.

**Comment by wf9a5m75** [comment]
Thank you for your suggestion. I have been kind of busy in couple of days. I will update this later.

**Comment by wf9a5m75** [comment]
Sorry, after the discussion with Mike, ionic team, I don't support the `@ionic-native/google-maps` anymore. I close this PR.

**Comment by wf9a5m75** [comment]
Since the problem has solved at once, I reopen this PR.

**Comment by wf9a5m75** [comment]
@ihadeed ping

**Pull request merged**